src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -104,6 +104,8 @@ public void save(AddressBook addressBook) throws StorageOperationException {
             marshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
             marshaller.marshal(toSave, fileWriter);
 
+        } catch (FileNotFoundException fnfe){
```

Comment:
This won't work as you expect. :(
The purpose is to check accidental file deletes and not check missing file initially.